### ISSUE TITLE
Wireframe structure

### ISSUE THREAD

**Issue by glassoctopus** [NSS-Evening-25-Cohort/e25-gitsub-tarnished-typers#59]
- [ ] Vanilla HTML, CSS, and JS only
- [ ] Your team should be using bootstrap
- [ ] each page should have a nav bar, a footer, creating a form to creates repositories
- [ ] Loop through your data to create a list of all the repos
- [ ] Ensure that every page has profile area and it is the same. Use JS to add the copy (content) to this. FOR STRETCH, you can try to get these other elements clickable or adding favorites, BUT only after you have completed MVP as a team

STRETCH GOAL:
Allow users to search through the repos using the search bar
Allow users to Star Repos
